### ISSUE TITLE
Heating operational tests doesn't allow for decimal points

### ISSUE THREAD

**Issue by melissasemcac** [fishbaugher/WAPLink-Issues#11]
Can we have the option to enter numbers with decimal points?  (i.e. 52 should be 5.2)
![image](https://github.com/user-attachments/assets/c0e2a6b8-a5c9-4e47-9196-f5aa4a87bbc6)

**Comment by BrianaParrBaker** [comment]
I agree that we need decimal points at least down to tenths.  The water heater operational tests does allow for decimals - furnace could be the same.
